package.json

```diff
@@ -66,6 +66,7 @@
     "react-transition-group": "^4.4.1",
     "react-virtual": "^2.3.2",
     "resize-observer-polyfill": "^1.5.1",
+    "sat": "^0.8.0",
```

Comment:
I just want @swissspidy's 👍  on [this package](https://github.com/jriecken/sat-js). It's MIT-licensed and seems respectable.